app/models/agents/scheduler_agent.rb

```diff
@@ -87,7 +87,7 @@ def working?
       true
     end
 
-    def check!
```

Comment:
No, this was correct.  SchedulerAgent is only invoked by lib/huginn_scheduler.rb via the `check!` method and the  `check` method is intentionally left undefined.